editor/modes/visual-editor/block.js

```diff
@@ -383,7 +406,7 @@ class VisualEditorBlock extends Component {
 								focus={ focus }
 								attributes={ block.attributes }
 								setAttributes={ this.setAttributes }
-								insertBlocksAfter={ onInsertBlocksAfter }
+								insertBlocksAfter={ ( blocks ) => onInsertBlocks( blocks, order + 1 ) }
```

Comment:
This seems like a component we might want to make pure, in which case we'd want to be careful about creating new function references on every render. Maybe this should point to a pre-bound instance method?

(Then again, we don't yet expose or take advantage of PureComponent, so this advice is premature)